studio/src/mas-placeholders.js

```diff
@@ -0,0 +1,1479 @@
+import { LitElement, html, nothing, css } from 'lit';
+import { repeat } from 'lit/directives/repeat.js';
+import StoreController from './reactivity/store-controller.js';
+import { ReactiveStore } from './reactivity/reactive-store.js';
+import Store from './store.js';
+import Events from './events.js';
+import { MasRepository } from './mas-repository.js';
+import './mas-folder-picker.js';
+import './filters/locale-picker.js';
+import { ROOT_PATH, DICTIONARY_MODEL_ID } from './constants.js';
+
+export function getDictionaryPath(folderPath, locale) {
+    if (!folderPath) {
+        return null;
+    }
+    if (!locale) {
+        return null;
+    }
+
+    const surface = folderPath;
+    const dictionaryPath = `${ROOT_PATH}/${surface}/${locale}/dictionary`;
+    return dictionaryPath;
+}
+
+class MasPlaceholders extends LitElement {
+    static styles = css`
```

Comment:
can you move styles to a css.js file?  when there are too many styles, it is hard to edit the file.